types/ali-oss/index.d.ts

```diff
@@ -915,6 +937,35 @@ declare class OSS {
      */
     deleteBucketCORS(name: string): Promise<OSS.NormalSuccessResponse>;
 
+    // policy operations
+    /**
+     * Adds or modify policy for a bucket.
+     */
+    putBucketPolicy(
+        name: string,
+        policy: OSS.BucketPolicy,
+        options?: OSS.RequestOptions
+    ): Promise<{
+        status: number,
+        res: OSS.NormalSuccessResponse,
+    }>;
+
+    /**
+     * Obtains the policy for a bucket.
+     */
+    getBucketPolicy(name: string, options?: OSS.RequestOptions): Promise<OSS.GetBucketPolicyResult>;
+
+    /**
+     * Deletes the policy added for a bucket.
+     */
+    deleteBuckyPolicy(
```

Comment:
Typo? `Bucky`

Reply:
Thank you for your review! I will fix the typo and test it as soon as possible.